boot.php

```diff
@@ -15,7 +23,8 @@
     throw new RuntimeException('You need to pass the ROM file name (Ex: drmario.rom)');
 }
 
-$filename = 'roms/'.$argv[1];
+    $filename = $argv[1];
```

Comment:
Just a little tab here! :stuck_out_tongue: 

I think you just need to squash your commits at one, now.

I'm sorry to bother you, man, and nice work BTW